ibm/data_source_ibm_is_images.go

```diff
@@ -180,6 +185,9 @@ func imageList(d *schema.ResourceData, meta interface{}) error {
 			"os":           *image.OperatingSystem.Name,
 			"architecture": *image.OperatingSystem.Architecture,
 		}
+		if image.File != nil && image.File.Checksums != nil && image.File.Checksums.Sha256 != nil {
+			l["checksum"] = *image.File.Checksums.Sha256
```

Comment:
image.File.Checksums.Sha256 != nil this is not required since  image.File.Checksums.Sha256 is a string not a complex structure

Reply:
Add respective documentations files datasource/resource with new attribute